### ISSUE TITLE
Flatpak Enve crashing...

### ISSUE THREAD

**Issue by dadoprom** [MaurycyLiebner/enve#131]
The steps to reproduce the crash:

1. draw a circle
2. fill > flat color> any color
3. make a key frame for fill color on timeline
4. go to next frame on timeline, change fill color
5. just click anywhere on the app or grab the cursor on timeline few times - enve should crash

this does not happen on latest .appimage version

**Comment by eszlari** [comment]
What's the output of

* the command line
* `journalctl -e`

?

**Comment by dadoprom** [comment]
[out.zip](https://github.com/MaurycyLiebner/enve/files/4621952/out.zip)

**Comment by eszlari** [comment]
The last few lines would have been enough! :smiley:

It crashes in `libgallium_dri.so`:
```
máj 13 14:31:17 185E0F1298C9 kernel: enve[19503]: segfault at 128 ip 00007f71fe2f4879 sp 00007fff38f146e0 error 4 in libgallium_dri.so[7f71fd91a000+cfd000]
```

The runtime provides a very recent version of Mesa (20.0.5 at the moment):
https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/blob/19.08/NEWS
I guess this is a regression in Mesa.

**Comment by eszlari** [comment]
@dadoprom Could post the ouput of

```
journalctl -g modesetting
```

(this time, you can post everything :smiley:)

**Comment by dadoprom** [comment]
[log.zip](https://github.com/MaurycyLiebner/enve/files/4623263/log.zip)

**Comment by eszlari** [comment]
Thanks. Maybe you could do a short video of the crash, I have a hard time reproducing the steps (not an animations guy).

**Comment by eszlari** [comment]
@dadoprom Have you applied any tearing fixes (`20-intel.conf` -> `TearFree`)?

**Comment by dadoprom** [comment]
@eszlari no TearFree, video in zip file

[crash.zip](https://github.com/MaurycyLiebner/enve/files/4624093/crash.zip)

**Comment by eszlari** [comment]
Since it's a driver related crash, I think this issue can be closed.

**Comment by eszlari** [comment]
@dadoprom I opened an issue on the mesa bug tracker:

https://gitlab.freedesktop.org/mesa/mesa/-/issues/2952

Hopefully, a future mesa version will fix your crash.

**Comment by eszlari** [comment]
@dadoprom A developer [said](https://gitlab.freedesktop.org/mesa/mesa/-/issues/2952#note_502741), this will be fixed in `mesa 20.0.7`.

**Comment by dadoprom** [comment]
@eszlari perfect, thanx  :1st_place_medal:
